1.8/administration/access-node/tunnel.md

```diff
@@ -0,0 +1,118 @@
+---
+nav_title: Access by Proxy
+post_title: Access by Proxy using DC/OS Tunnel
+menu_order: 10
+---
+
+When developing applications on DC/OS, you may find it helpful to access your cluster from your local machine via SOCKS proxy, HTTP proxy, or VPN. For instance, you can work from your own development environment and immediately test against your DC/OS cluster.
+
+# SOCKS
+DC/OS Tunnel can run a SOCKS proxy over SSH to the cluster. SOCKS proxies work for any protocol, but your application must be configured to use the proxy, which runs on port 1080 by default. Access URLs by appending the `mydcos.directory` domain.
+
+# HTTP
+Run the HTTP proxy on your local port 80 to tunnel HTTP requests without modifying your application. Access URLs by appending the `mydcos.directory` domain. You can also [use DNS SRV records as if they were URLs](#srv). The HTTP proxy requires root access to use port 80, but can be configured to use a different port.
+
+<a name="srv"></a>
+## SRV Records
+A SRV DNS record is a mapping from a name to a IP/port pair. DC/OS creates SRV records in the form `_<port-name>._<service-name>._tcp.marathon.mesos`. The HTTP proxy exposes these as URLs. This feature can be useful for communicating with DC/OS services.
+
+# VPN
+DC/OS Tunnel provides you with full access to the DNS, masters, and agents from within the cluster. OpenVPN requires root privileges to configure these routes.
+
+# Using DC/OS Tunnel
+
+## Prerequisites
+* DC/OS 1.8+
+* Docker
+* Requires [SSH access](/1.8/administration/access-node/sshcluster/) (key authentication only).
+* For VPN functionality, you must have [the OpenVPN client](https://openvpn.net/index.php/open-source/downloads.html) installed.
+* The DC/OS Tunnel package must be installed: run `dcos package install tunnel-cli --cli` from the DC/OS CLI.
+
+**Note:** Only Linux and OS X are supported currently.
+
+##  SOCKS
+Run the following command from the DC/OS CLI:
+
+```
+$ dcos tunnel socks
+```
+
+Configure your application to use the proxy on port 1080: `127.0.0.1:1080`
+
+### Port Forwarding
+The SOCKS proxy works by port forwarding. Append `.mydcos.directory` to the end of your domain when you enter commands. For instance, `http://example.com:1080/?query=hello` becomes `http://example.com.mydcos.directory:1080/?query=hello`.
```

Comment:
Oops, I just realized, the port you access in the URL is unrelated to the port that the SOCKS proxy is running on. The SOCKS proxy port is local (e.g. on your laptop), and the URL port is remote (e.g. on your DC/OS cluster). So this port shouldn't be the SOCKS port to avoid being misleading. Sorry for flipping back and forth on this point.

The same goes for the Port Forwarding under the HTTP part. They should both be some arbitrary port